Update ns-winnt-claim_security_attribute_v1.md (#1313)
MicrosoftDocs/sdk-api: sdk-api-src/content/winnt/ns-winnt-claim_security_attribute_v1.md

Pointer to an array of <b>ValueCount</b> members where each member is  an octet string of type <a href="/windows/win32/api/winnt/ns-winnt-claim_security_attribute_octet_string_value">CLAIM_SECURITY_ATTRIBUTE_OCTET_STRING_VALUE</a>.

## -remarks

> The field value type suggests that the value can be an Octet string or a SID. However the <a href="https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-adts/252d7e10-eaf8-44e9-8b8d-205b384f5782">Directory Services documentation for claims entries</a> specifies that effective possible data types for claims are only Int64, UInt64, UnicodeString and Boolean.

Reviewer: ```suggestion
The field value type indicates that the value can be an octet string or a SID. However, the [Directory Services documentation for claims entries](/openspecs/windows_protocols/ms-adts/252d7e10-eaf8-44e9-8b8d-205b384f5782) specifies that effective possible data types for claims are limited to Int64, UInt64, UnicodeString, and Boolean.
```
